src/plugins/index_pattern_management/public/components/index_pattern_table/index_pattern_table.tsx

```diff
@@ -183,7 +183,7 @@ export const IndexPatternTable = ({ canSave, history }: Props) => {
   const columns = [
     {
       field: 'title',
-      name: 'Pattern',
+      name: 'Title',
```

Comment:
Do we have insight or if this favorable or not?

Reply:
@KrooshalUX requested this change. She mentioned to be consistent with save object page.

Reply:
@kristenTian @KrooshalUX - We discussed in the brainstorming meeting today - the "title" reference in the saved object table is a bit misleading, and "pattern" or "index pattern" is the more accurate term. One potential solution in the future would be to disambiguate the index pattern saved object title from the pattern itself: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2649